assets/react/v3/shared/components/modals/AddLessonModal.tsx

```diff
@@ -0,0 +1,241 @@
+                    content={<span css={styles.durationContent}>{__('hour', 'tutor')}</span>}
+                    contentPosition='right'
+                    label={__('Duration', 'tutor')}
+                    placeholder='0'
+                    showVerticalBar={false}
+                  />
+                )}
+              />
+              <Controller
+                name='duration_min'
+                control={form.control}
+                render={(controllerProps) => (
+                  <FormInputWithContent
+                    {...controllerProps}
+                    type='number'
+                    content={<span css={styles.durationContent}>{__('min', 'tutor')}</span>}
+                    contentPosition='right'
+                    placeholder='0'
+                    showVerticalBar={false}
+                  />
+                )}
+              />
+              <Controller
+                name='duration_sec'
+                control={form.control}
+                render={(controllerProps) => (
+                  <FormInputWithContent
+                    {...controllerProps}
+                    type='number'
+                    content={<span css={styles.durationContent}>{__('sec', 'tutor')}</span>}
+                    contentPosition='right'
+                    placeholder='0'
+                    showVerticalBar={false}
+                  />
+                )}
+              />
+            </div>
+
+            <div css={styles.uploadAttachment}>
+              <span css={styles.uploadLabel}>Exercise Files</span>
+              <Button
+                icon={<SVGIcon name='attach' height={24} width={24} />}
+                variant='secondary'
+                buttonContentCss={css`
+                  justify-content: center;
+                `}
+              >
+                Upload Attachment
+              </Button>
+            </div>
+
+            <div css={styles.lessonPreview}>
+              <Controller
+                name='lesson_preview'
+                control={form.control}
+                render={(controllerProps) => (
+                  <FormSwitch {...controllerProps} label={__('Lesson preview', 'tutor')} helpText='Show preview' />
+                )}
+              />
+            </div>
+          </div>
+        </div>
+      </div>
+    </ModalWrapper>
+  );
+};
+
+export default AddLessonModal;
+
+const styles = {
+  wrapper: css`
+    width: 1035px;
+    margin: 0 auto;
+    display: grid;
+    grid-template-columns: 1fr 395px;
+    height: 100%;
+  `,
+  lessonInfo: css`
+    padding-block: ${spacing[24]};
+    padding-left: ${spacing[24]};
+    padding-right: ${spacing[64]};
+    display: flex;
+    flex-direction: column;
+    row-gap: ${spacing[24]};
+    align-self: start;
+    position: sticky;
+    top: 0;
+  `,
+  rightPanel: css`
+    border-left: 1px solid ${colorTokens.stroke.divider};
+    display: flex;
+    flex-direction: column;
+    row-gap: ${spacing[16]};
```

Comment:
Do we really need `row-gap`?
```suggestion
    gap: ${spacing[16]};
```